### DISCUSSION TITLE
Scelta installazione per Home Assistant

### DISCUSSION THREAD

**Discussion by officialdavide** [indomus/forum#500] [HUB: Home Assistant]
Ciao,
ho un Raspberry 4 GB di RAM e devo installare Home Assistant.

Al tempo stesso sul Raspberry devo far girare applicativi Java, alcuni database di MySQL e molto probabilmente alcuni script per automazioni: tutto fuori HomeAssistant.

Per garantire più stabilità al mio sistema, uso un vecchio hard-disk di qualche centinaio di GB, collegato via USB.

Volevo installare Home Assistant con Docker ma temo per i cambiamenti futuri, come comunicazione di alcuni mesi fa.

Che cosa mi consigliate?

**Comment by snakuzzo** [comment]
I cambiamenti futuri che erano usciti fuori e a cui ti riferisci non erano relativi a docker, ma al tipo di installazione HA Supervised su Docker.
L'installazione di Core su Docker è strasupportata e lo sarà sempre.

**Comment by KC-inDomus** [comment]
personalmente ti consiglio Home Assistant Core @Docker.
